lib/debezium/keys.go

```diff
@@ -70,8 +100,7 @@ func parsePartitionKeyStruct(keyBytes []byte) (map[string]any, error) {
 	}
 
 	var pkStruct map[string]any
-	err := json.Unmarshal(keyBytes, &pkStruct)
-	if err != nil {
+	if err := json.Unmarshal(keyBytes, &pkStruct); err != nil {
```

Comment:
It would be nice to change the error message on line 108 to something more specific since it's the same message as line 99.